docs/content/stable/develop/drivers-orms/nodejs/sequelize.md

```diff
@@ -189,6 +210,71 @@ Employees Details:
 ]
 ```
 
+#### Specify load balance properties
+
+You can specify load balance properties using environment variables or connection parameters.
+
+For information on all load balance modes, see [Node type-aware load balancing](../../smart-drivers/#node-type-aware-load-balancing).
+
+##### Environment variables
+
+The following table summarizes the environment variables available for configuring the YugabyteDB node-postgres smart driver:
```

Comment:
@ashetkar These Env variables are common to the driver not just sequelize? Why is this table not here https://docs.yugabyte.com/stable/develop/drivers-orms/nodejs/yugabyte-node-driver/#step-2-set-up-the-database-connection